TSQL_Parser/TSQL_Parser/TSQLKeywords.cs

```diff
@@ -154,6 +154,7 @@ public struct TSQLKeywords
 		public static readonly TSQLKeywords OR = new TSQLKeywords("OR");
 		public static readonly TSQLKeywords ORDER = new TSQLKeywords("ORDER");
 		public static readonly TSQLKeywords OUTER = new TSQLKeywords("OUTER");
+		public static readonly TSQLKeywords OUTPUT = new TSQLKeywords("OUTPUT");

```

Comment:
It's painful, but I don't think I currently want these words added as keywords.

They are not currently part of the [Reserved Keywords](https://docs.microsoft.com/en-us/sql/t-sql/language-elements/reserved-keywords-transact-sql?view=sql-server-ver15) list, but are on the future reserved words list.

These statements below are currently legal, without escaping, and adding these words to the keywords list will eventually cause the SELECT and MERGE parsers problems if similarly used in one of those statements without escaping:

`create table output (id int); create table using (id int);`

Management Studio syntax color coding does show blue for these words, so in [TSQL.Color](https://github.com/bruce-dunwiddie/tsql-color/blob/master/TSQL.Color/TSQL.Color/SQLColorizer.cs) I had to create a pseudo keywords list to mimic that functionality.

I think we may have to add a helper class or something to ease the coding of statement parsers that depend on these words, but without modifying the token type returned.

Reply:
Great catch, this is not something I had anticipated. I merely went to the documentation and searched the page for these keywords without realizing that they were listed as **Future Keywords**. I will comment them out from `TSQLKeywords.cs`.

Reply:
I've added the class TSQLFutureKeywords in commit [a0e8ca1](https://github.com/bruce-dunwiddie/tsql-parser/commit/a0e8ca1390738b3257e3714f298311b8862cfa9f) to the develop branch.

To ease your local merge, I did not add the class to the project and commented out the new method in TSQLToken that you should be able to use within your statement parser.

We can add more of these problematic words to this class down the road.

Reply:
* Merged devlop into feature/add-merge-statement branch
* Added TSQLFutureKeywords class to project
* Created TSQLFutureKeyword token class to distinguish from TSQLKeyword token and TSQLIdentifier token.
* Added test for create table examples from comment above
  * OUTPUT and USING are of type TSQLFutureKeyword in MERGE statements but TSQLIdentifier elsewhere
* Added branch to TSQLWithClauseStatementParser to allow CTE's within MERGE statements